src/main/scala/miss/trafficsimulation/actors/AreaActor.scala

```diff
@@ -60,25 +60,36 @@ class AreaActor extends FSM[State, Data] {
         throw new RuntimeException("Some cars are missing: " + (beforeCarsCount - outgoingTraffic.size - afterCarsCount))
       }
 
-      val messagesSent = mutable.Map(area.actorsAndRoadIds.map({
+      // sending outgoing traffic
+      val messagesToSend = mutable.Map(area.outgoingActorsAndRoadIds.map({
         case (a: ActorRef, r: RoadId) => (a, r) -> false
       }): _*)
       outgoingTraffic groupBy {
         case (actorRef, roadId, _) => (actorRef, roadId)
       } foreach {
         case ((actorRef, roadId), list) =>
           log.debug(s"Sending to $actorRef; roadId: $roadId; traffic: $list")
-          messagesSent((actorRef, roadId)) = true
+          messagesToSend((actorRef, roadId)) = true
```

Comment:
This name should not change. I was putting here which messages I've already sent - this map was containing the following mapping:
```
(actor, roadId) -> hasOutgoingTraffic (so the message was sent during simulation)
```
So I think we should change back the name or change it to sth more meaningful. But not `messagesToSend`.